huaweicloud/services/as/data_source_huaweicloud_as_groups.go

```diff
@@ -0,0 +1,321 @@
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"current_instance_number": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"desire_instance_number": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"min_instance_number": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"max_instance_number": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"cool_down_time": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"lb_listener_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"lbaas_listeners": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"pool_id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"protocol_port": {
+										Type:     schema.TypeInt,
+										Computed: true,
+									},
+									"weight": {
+										Type:     schema.TypeInt,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"availability_zones": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+						"networks": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"ipv6_enable": {
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
+									"ipv6_bandwidth_id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"security_groups": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"create_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"vpc_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"detail": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"is_scaling": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+						"health_periodic_audit_method": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"health_periodic_audit_time": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"health_periodic_audit_grace_period": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"instance_terminate_policy": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"notifications": {
```

Comment:
`notifications` and `lb_listener_id` are marked as deprecated, so we can delete it directly in the data source

Reply:
done